pkgs/development/ocaml-modules/protoc/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, buildOcaml, ppx_deriving_protobuf }:
+
+buildOcaml rec {
+  name = "protoc";
```

Comment:
The library name is `ocaml-protoc`. So the derivation name should be `ocaml${ocaml.version}-ocaml-protoc-${version}`.